### PR TITLE
Multiple minor fixes

### PR DESCRIPTION
**Pull request by ankitjavalkar**
- Display a message when the question paper exists but has no questions
- Avoid logout after encountering no quiz/no questions
- Remove incorrect logout message in complete

### CODE REVIEW

File: yaksh/templates/yaksh/complete.html [hunk 1]
```diff
@@ -32,5 +32,5 @@
 		<center><h2> Good bye! </h2></center>
 		<center><h4> {{message}} </h4></center>
 		<br><center><h4>You may now close the browser.</h4></center><br>
-	<center><a href="{{URL_ROOT}}/exam/" id="login_again"> Login Again </a></center>
+	<center><a href="{{URL_ROOT}}/exam/" id="go_to_home"> Go To Home </a></center>
```
Comment (prabhuramachandran) [inline yaksh/templates/yaksh/complete.html]: I don't like "Go To Home"

Comment (prabhuramachandran) [inline yaksh/templates/yaksh/complete.html]: Maybe "Click here to return to Yaksh."

Comment (ankitjavalkar) [inline yaksh/templates/yaksh/complete.html]: OK, how about just "Home" or "Dashboard", I just thought "Click here to return to Yaksh" seemed a bit too long for a Button label

Comment (prabhuramachandran) [inline yaksh/templates/yaksh/complete.html]: LGTM.

**Pull request merged**